test/integration/functional_test.go

```diff
@@ -591,7 +606,13 @@ func validateCacheCmd(ctx context.Context, t *testing.T, profile string) {
 			if err != nil {
 				t.Errorf("failed to get images by %q ssh %v", rr.Command(), err)
 			}
-			if !strings.Contains(rr.Output(), "0184c1613d929") {
+			var pauseID string
+			if Arm64Platform() {
```

Comment:
Can you make this a switch statement or map containing hashes for different architectures? I feel like we shouldn't hard-code yet another architecture here, when there will certainly be others in the near-term support horizon, such as riscv.

Reply:
Done